scripts/runner.go

```diff
@@ -46,3 +49,10 @@ func GetRepoRootDir() (string, error) {
 	dirPath := filepath.Dir(filepath.Dir(filename))
 	return dirPath, nil
 }
+
+func ExecuteScript(t *testing.T, scriptPath string) []byte {
```

Comment:
Better call this "AssertExecuteScript" to indicate its doing an assertion.

Reply:
Changed.